src/lookup-table/models/LookupTable.model.ts

```diff
@@ -64,26 +64,26 @@ const LookupTableModel = () => {
       const result = await DBConnect.gqlQuery(
         `
           query countStructureRowsByTableName($name: String!) {
-            lookupTables(condition: {name: $name}) {
+            dataTables(condition: {name: $name}) {
               totalCount
             }
           }
         `,
         { name: lookupTableName }
       )
 
-      return result.lookupTables.totalCount as number
+      return result.dataTables.totalCount as number
     } catch (error) {
       throw error
     }
   }
 
   const createTable = async ({
-    name: tableName,
+    tableName,
     fieldMap: fieldMaps,
   }: LookupTableBase): Promise<boolean> => {
     try {
-      const text = `CREATE TABLE lookup_table_${tableName}
+      const text = `CREATE TABLE data_table_${tableName}
```

Comment:
Should we also receive the prefix as param?

Reply:
Yes, we probably should. I'll do it now.